### ISSUE TITLE
Fix active.js

### ISSUE THREAD

**Issue by hvarday** [rahulkapoor90/VITacademics-Enhancement-Suite#13]
The script is currently disabled. It works fine, but causes other scripts to malfunction when it is enabled.

**Comment by armudgal** [comment]
Is someone working on it? I was thinking to start playing with this

**Comment by rahulkapoor90** [comment]
Yes. No one is working on it yet so you can continue working on it.
